### ISSUE TITLE
Phylogeny to accompany data

### ISSUE THREAD

**Issue by A-Friendly-Fish** [alarconvv/PaleoSize#1]
Hey, I love the data you're compiling.  What project are you working on?  I don't want to accidentally steal any of your ideas or discoveries.  But if it's possible, I'd like to test a new phylogenetic comparative method I'm developing on this great data you're compiling.  Do you have any phylogenies to go along with the data? https://github.com/alarconvv/PaleoSize/tree/master/Databases/Allan%20et%20al%202014 for example has the excel data sheet and the original paper, but no phylogeny.  Anyways,  I'd love to collaborate if you're interested.

**Comment by alarconvv** [comment]
Hi

I'm sorry for not getting back to you before. I am currently evaluating
body mass estimation as a machine-learning problem. I want to explore if I
can improve current models. Then, gathering every model in an app. Finally,
evaluate these hypotheses over the phylogeny. Currently, I am collecting
data, but I have done some evaluations in bipedal no-avian therapods.

The email you sent me is encrypted, so I am not sure if you will get this
reply. Could you reply it to ***@***.***?

My best
Viviana

Happy holidays

**Comment by A-Friendly-Fish** [comment]
Yes. This is very weird, I posted the original comment on github
https://github.com/alarconvv/PaleoSize/issues/1   I wasn't trying to send
anything encrypted.  My email is afriendlyfish0 AT gmail.  So hopefully you
can contact me there, the email I got from you is all ***s, so I can't read
it unfortunately.  Sorry for all the confusion!

I'd love to learn more about your research and tell you about mine so we
can figure out if there is a collaboration opportunity or not.   If you are
interested, I'd love to video chat.

Thanks!
